src/test/scala/scynamo/ScynamoCodecProps.scala

```diff
@@ -92,9 +92,15 @@ class ScynamoCodecProps extends Properties("ScynamoCodec") {
       decodeAfterEncodeIsIdentity(Duration.fromNanos(value): Duration)
     }
 
-  propertyWithSeed("decode.encode === id (year month)", propertySeed) = Prop.forAll { value: YearMonth =>
-    decodeAfterEncodeIsIdentity(value)
-  }
+  propertyWithSeed("decode.encode === id (java duration)", propertySeed) =
+    Prop.forAll(Gen.chooseNum[Long](-9223372036854775807L, 9223372036854775807L)) { value: Long =>
```

Comment:
I'd love to avoid using magic values in the code.
```suggestion
    Prop.forAll(Gen.chooseNum[Long](Long.MinValue, Long.MaxValue)) { value: Long =>
```